components/Sidebar/index.jsx

```diff
@@ -0,0 +1,93 @@
+import styles from './sidebar.module.css';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+
+const sidebarMenuOptions = [
```

Comment:
Kudos for making the scalable sidebar :fire: 

Just transfer this `sidebarMenuOptions` to constants/data folder